### PR TITLE
Add scripts/generate-snippets.vsh

### PR DESCRIPTION
**Pull request by Zhomart**
V Script is a script written in vlang. It can be executed using: `$ ./scripts/generate-snippets.vsh`.

I verified that both python and vsh generates the same content.

Hi @elliotchance, please take a look at this PR. I'm not sure if you were planning to migrate python files to V. You can close this PR if you don't need it.

### CODE REVIEW

File: scripts/generate-snippets.vsh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env -S v
```
Comment (elliotchance) [inline scripts/generate-snippets.vsh:1]: Thanks! Can you unwrap this from a shell script (so its a raw V file) and update the Makefile accordingly.

File: .gitignore [hunk 2]
```diff
@@ -4,3 +4,5 @@ vsql-server
 *.vsql
 .vscode
 .vlang_tmp_build/
+
+scripts/generate-snippets
```
Comment (elliotchance) [inline .gitignore:8]: Was this just left over?

**Review by elliotchance — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (Zhomart) [reply .gitignore:8]: Apparently .vsh generates this binary.  I think we should ignore it.

File: scripts/generate-snippets.vsh (hunk 1 above)
Comment (Zhomart) [reply scripts/generate-snippets.vsh:1]: Do you prefer raw "v" file or a v script file?

V script docs: https://github.com/vlang/v/blob/master/doc/docs.md#cross-platform-shell-scripts-in-v

Comment (Zhomart) [reply scripts/generate-snippets.vsh:1]: I've updated the `Makefile`

**Review by elliotchance — APPROVED** [review]

**Comment by elliotchance** [comment]
I'd never heard of V scripts, but it makes sense now. Thanks for this!

**Comment by elliotchance** [comment]
If your interested in working on something similar, https://vsql.readthedocs.io/en/latest/v-client-library.html#v-module is written manually right now. The snippets you refactored on are what provide the content for each one, however, if we can generate that page from reading all the public elements in V that would be so much better.

**Comment by Zhomart** [comment]
> If your interested in working on something similar, https://vsql.readthedocs.io/en/latest/v-client-library.html#v-module is written manually right now. The snippets you refactored on are what provide the content for each one, however, if we can generate that page from reading all the public elements in V that would be so much better.

Created a proposal: https://github.com/elliotchance/vsql/pull/154

**Pull request merged**